stack.yaml

```diff
@@ -9,3 +9,4 @@ extra-deps:
 - repline-0.2.0.0
 - async-pool-0.9.0.2
 - serialise-0.2.1.0
+- Win32-2.5.4.1@sha256:e623a1058bd8134ec14d62759f76cac52eee3576711cb2c4981f398f1ec44b85
```

Comment:
It looks like version [`2.8.2.0` of Win32](http://hackage.haskell.org/package/Win32-2.8.2.0) is out, so we might want to try to build with it (since this 2.5 version is 6 months old)

There's a bug open upstream about Win32 support in Turtle ([here](https://github.com/Gabriel439/Haskell-Turtle-Library/issues/323)). It's not affecting us, but it probably means that some versions of Win32 don't build with some versions of Turtle.

Reply:
Settings this to `- Win32-2.8.2.0` gives

> Error: While constructing the build plan, the following exceptions were encountered:
> 
> In the dependencies for Cabal-2.2.0.1:
>     Win32-2.8.2.0 from stack configuration does not match >=2.2.2 && <2.7  (latest matching version
>                   is 2.6.2.0)
> needed due to spago-0.6.0.0 -> Cabal-2.2.0.1
> 
> In the dependencies for directory-1.3.1.5:
>     Win32-2.8.2.0 from stack configuration does not match >=2.2.2 && <2.7  (latest matching version
>                   is 2.6.2.0)
> needed due to spago-0.6.0.0 -> directory-1.3.1.5
> 
> In the dependencies for process-1.6.3.0:
>     Win32-2.8.2.0 from stack configuration does not match >=2.2 && <2.7  (latest matching version
>                   is 2.6.2.0)
> needed due to spago-0.6.0.0 -> process-1.6.3.0
> 
> In the dependencies for turtle-1.5.12:
>     Win32-2.8.2.0 from stack configuration does not match >=2.2.0.1 && <2.6  (latest matching
>                   version is 2.5.4.1)
> needed due to spago-0.6.0.0 -> turtle-1.5.12
> 
> Some different approaches to resolving this:
> 
>   * Set 'allow-newer: true' in C:\sr\config.yaml to ignore all version constraints and build anyway.
> 
>   * Consider trying 'stack solver', which uses the cabal-install solver to attempt to find some
>     working build configuration. This can be convenient when dealing with many complicated
>     constraint errors, but results may be unpredictable.
> 
>   * Recommended action: try adding the following to your extra-deps
>     in C:\VisualStudio_Projects\purescript\spago\stack.yaml:
> 
> Win32-2.6.2.0@sha256:515518df0728294c99822c65b04be10a242933f40240b125c23469f45c8b11da

Reply:
Makes sense. How about trying `Win32-2.6.2.0`?

Reply:
> Error: While constructing the build plan, the following exceptions were encountered:
> 
> In the dependencies for turtle-1.5.12:
>     Win32-2.6.2.0 from stack configuration does not match >=2.2.0.1 && <2.6  (latest matching
>                   version is 2.5.4.1)
> needed due to spago-0.6.0.0 -> turtle-1.5.12
> 
> Some different approaches to resolving this:
> 
>   * Set 'allow-newer: true' in C:\sr\config.yaml to ignore all version constraints and build anyway.
> 
>   * Consider trying 'stack solver', which uses the cabal-install solver to attempt to find some
>     working build configuration. This can be convenient when dealing with many complicated
>     constraint errors, but results may be unpredictable.
> 
>   * Recommended action: try adding the following to your extra-deps
>     in C:\VisualStudio_Projects\purescript\spago\stack.yaml:
> 
> Win32-2.5.4.1@sha256:e623a1058bd8134ec14d62759f76cac52eee3576711cb2c4981f398f1ec44b85
> 
> Plan construction failed.